tests/karma/unit/services/enketo-prepopulation-data.js

```diff
@@ -35,10 +35,12 @@ describe('EnketoPrepopulationData service', function() {
             '<user>' +
               '<name/>' +
             '</user>' +
-            '<location>' +
-              '<lat/>' +
-              '<long/>' +
-            '</location>' +
+            '<meta>' +
+              '<location>' +
+                '<lat/>' +
+                '<long/>' +
+              '</location>' +
+            '</meta>' +
```

Comment:
A quick scan of medic-projects/lg-uganda/forms confirms your theory, so stick with the test change.